frontend/src/metabase/lib/entities.js

```diff
@@ -664,3 +677,13 @@ export const notify = (opts = {}, subject, verb) =>
 
 export const undo = (opts = {}, subject, verb) =>
   merge({ notify: { subject, verb, undo: true } }, opts || {});
+
+export async function entityCompatibleQuery(entityQuery, dispatch, endpoint) {
+  const request = entityQuery === EMPTY_ENTITY_QUERY ? undefined : entityQuery;
+  const action = dispatch(endpoint.initiate(request, { forceRefetch: true }));
+  try {
+    return await action.unwrap();
+  } finally {
+    action.unsubscribe();
+  }
```

Comment:
```suggestion
  const action = dispatch(endpoint.initiate(request, { forceRefetch: true, subscribe: false }));
  return await action.unwrap();
```
I think this does the same thing.

Reply:
When I do that I no longer get results in `unwrap`